services/apps/alcs/src/providers/typeorm/migrations/1687557623816-local_gov_name_edits.ts

```diff
@@ -5,9 +5,10 @@ export class localGovNameEdits1687557623816 implements MigrationInterface {
     public async up(queryRunner: QueryRunner): Promise<void> {
         await queryRunner.query(`
           UPDATE "alcs"."application_local_government" SET "name" = 'Village of Masset' WHERE "name" = 'Village of Massett';
+          UPDATE "alcs"."application" SET "local_government_uuid" = '92961db6-b74c-460b-bbad-e285398fa491' WHERE "local_government_uuid" = '33aa1f7d-3b65-4ed5-badf-11dafb0b2789';
```

Comment:
What about the planning review and covenants? Is 92961db6-b74c-460b-bbad-e285398fa491 is active one?

Reply:
Fixed!